WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+  
+Note that the process to join and leave teams is out of scope of this WEP and will be handled in a future WEP.
+
+At a very high level, the WEP submission process includes the following stages:
+
+#. `Pre-proposal`_ — Someone has an idea and they start collecting early input and feedback to see if it is within the scope of a WEP. This can be done informally, publicly or privately.
+
+#. `Draft submission and pre-review`_ — The WEP author writes a rough draft of the WEP and submits it in a pull request (PR) to the WEPs repository. The draft is pre-reviewed by an editor for style, formatting, and eligibility for the WEP process. If the WEP is pre-approved, it is accepted for discussion and the 30-day review period begins.
+
+#. `Community review, discussion, and updates`_ — During the review period, the WEP draft is discussed, improved, and updated based on incoming feedback from the community. Feedback is welcome from the whole community, as long as it arrives within the review period. 
+
+#. `Final review and resolution`_ — At the end of the review period, the WEP is examined for **lazy consensus**. A lazy consensus means a situation where the community either provided feedback, approved the WEP explicitly, or ignored the WEP altogether. In some cases where consensus wasn't reached, the review period might be extended, but never shortened.
+
+#. `Implementation`_ — If the WEP is accepted, the processes or policies described in the WEP are implemented by the CoreOps team, the WEP author, or specific community members as defined in the WEP.
+
+The following sections describe each stage in more detail.
+
+Pre-proposal
+............
+
+The WEP process begins with a new idea for Write the Docs. It is highly recommended that a single WEP contain a single key proposal or new idea. 
+
+To help prepare the WEP draft, we encourage community members to discuss the idea publicly, for example in the Slack group, and gather initial community feedback before writing a full WEP. Public discussions on pre-proposals also help to make sure the idea is applicable to the entire community and not just to the author.
+
+A WEP can be co-authored by more than one community member, but we recommend keeping the number of co-authors small and choosing a representative who will submit the PR to the WEPs repository on behalf of the co-authors.
+
+Draft submission and pre-review
+...............................
+
+A draft WEP must be submitted in a GitHub pull request to the `writethedocs/weps <https://github.com/writethedocs/weps>`_ repository, in the ``drafts/`` directory. If you need help with working with GitHub or creating a PR, please feel free to send an e-mail to support@writethedocs.org, and the Write the Docs team will help you get started.
```

Comment:
Maybe "To submit a draft WEP, set up a GitHub pull request on the ..."

Reply:
I think the passive voice here is again probably better, since we're trying to define policy, not necessarily provide a user guide. Though we are speaking to the users a bit, so I think it's blurry. I'm thinking we probably just need to create a "how to submit a WEP" guide, that would exist outside of this policy, since we *really* don't want people to have to read this whole policy in order to submit a WEP.

Reply:
Should we make it an explicit thing in the WEP? Having a Getting Started doc shouldn't be too hard to extrapolate out of this PR, but not sure if we should mention it here.

Reply:
> we probably just need to create a "how to submit a WEP" guide, that would exist outside of this policy, since we really don't want people to have to read this whole policy in order to submit a WEP.

I think that addresses the crux of my concerns -- I'm reading this from the PoV of someone who might submit WEPs.

Reply:
Thinking big picture: we need more volunteers. Few people understand how to volunteer to help, beyond what we do at conferences. While I understand the need for this WEP process, I think this feels like another barrier to entry for new contributors. 

A simplfied "how to submit a WEP" guide IMO would help.

Reply:
Created a new issue #2 to handle a how-to guide, since it'll be separate from this WEP